<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IP_SettingToolWid</class>
 <widget class="QWidget" name="IP_SettingToolWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="timeBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>时间间隔</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>命令模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>级联数量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="serialBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>串口设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="devNumBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmdBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>单命令</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双命令</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="serialLab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>COM0  未打开</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>33</number>
   </property>
  </widget>
  <widget class="QComboBox" name="timeBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>500ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1000ms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1500ms</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="vBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>设备版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="vBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>V1版本</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V3版本</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
